cmake/modules/FindFlatBuffers.cmake

```diff
@@ -46,7 +46,7 @@ if(FLATBUFFERS_FOUND)
   set(FLATBUFFERS_INCLUDE_DIRS ${FLATBUFFERS_INCLUDE_DIR} ${FLATBUFFERS_MESSAGES_INCLUDE_DIR})
 
   if(NOT TARGET flatbuffers::flatbuffers)
-    add_library(flatbuffers::flatbuffers UNKNOWN IMPORTED)
+    add_library(flatbuffers::flatbuffers INTERFACE IMPORTED)
```

Comment:
What's the difference between these two?

Reply:
an unknown imported target is done when you dont know if the lib is static/shared/executable. these states are intended to have a file on disk. An interface doesnt require a file to be physically present, but allows to propagate other properties of targets when linked.

Reply:
Got, so `UNKNOWN` lib/exe on disk, `INTERFACE` instead to propagate properties

Reply:
not exactly. https://cmake.org/cmake/help/latest/command/add_library.html#imported-libraries for some info